src/Core/Algorithms/Factory/HardCodedAlgorithmFactory.cc

```diff
@@ -95,6 +96,7 @@ using namespace SCIRun::Core::Algorithms::DataIO;
 using namespace SCIRun::Core::Algorithms::Math;
 using namespace SCIRun::Core::Algorithms::BrainStimulator;
 using namespace SCIRun::Core::Algorithms::Math;
+using namespace SCIRun::Core::Algorithms::Converters;
```

Comment:
@mhansen1 You're missing the line to actually connect the module and algorithm. Running your test network results in an error leading to this file: 

ERROR: Null algorithm object, make sure AlgorithmFactory knows about this module's algorithm types.